documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -203,14 +198,12 @@ This option sets the country code for your wireless network.
 
 image::images/raspi-adv.png[raspi-config advanced options]
 
-NOTE: The options documented here will change depending on the model of Raspberry Pi that you're using, and whether you're using the Wayland or X11 backend. In the current version of Raspberry Pi OS Bookworm, the Raspberry Pi 4, Pi 400 and Pi 5 use Wayland by default; other models of Raspberry Pi use X11 by default.
-
 [[expand-filesystem]]
 ===== Expand filesystem
 
-This option will expand your installation to fill the whole SD card, giving you more space to use for files. You will need to reboot the Raspberry Pi to make this available. 
+Expands your installation to fill the entire storage device, giving you more space to use for files. You must reboot the Raspberry Pi to make this available.
```

Comment:
Raspberry Pi OS _should_ do this automatically, during its first-boot process. I think the option is only in `raspi-config` as a fail-safe fallback? (or if you e.g. use the "SD Card Copier" to clone your OS to a larger storage-medium)

Reply:
Worth mentioning for sure.